kahypar/partition/coarsening/policies/fixed_vertex_acceptance_policy.h

```diff
@@ -56,16 +56,6 @@ class FixedVertexAcceptancePolicy : public meta::PolicyBase {
     const HypernodeWeight node_weight = hg.isFixedVertex(v) ? 0 : hg.nodeWeight(v);
     return fixed_vertex_part_weight + node_weight <= max_allowed_fixed_vertex_part_weight;
   }
-
```

Comment:
@kittobi1992 ?

Reply:
The coarsening algorithms are now based on the number of free vertices, therefore the additional test in FixedVertexAcceptancePolicy is not required anymore.